sql/parser/sql.y

```diff
@@ -2807,6 +2808,14 @@ func_expr_common_subexpr:
 | GREATEST '(' expr_list ')' { unimplemented() }
 | LEAST '(' expr_list ')' { unimplemented() }
 
+func_or_variable:
```

Comment:
I'm in favor of removing the variable treatment for all these functions. The "modernized" approach does that.